pom.xml

```diff
@@ -1114,6 +1114,12 @@
                 <version>3.6.1</version>
```

Comment:
> TestingKafka is used by multiple modules for testing.

What modules?

If you are copying it can you please remove testing kafka from other place and reuse your testing kafka?

Reply:
That's done in the commit "Use TestingKafka from presto-testing". Should I squash this one with the previous 2 commits?

Reply:
Yes, let squash them.